CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased] 
 ### Added
 - Query parameters for sorting by given field in ascending and descending order [#111](https://github.com/IN-CORE/incore-services/issues/111)
+- Get user group function to read and parse user groups from headers [#120](https://github.com/IN-CORE/incore-services/issues/120)
+
+### Changed
+- Remove LdapClient and refactor the Authorizer class along with its interface to take in userGroups parameter [#118](https://github.com/IN-CORE/incore-services/issues/118)
+- Add userGroups parameter in Semantics Controller, Space Controller, DFR3 Controller, Data Controller, Hazard Controller and modify internal calls to support userGroups parameter [#124](https://github.com/IN-CORE/incore-services/issues/124) [#123](https://github.com/IN-CORE/incore-services/issues/123) [#122](https://github.com/IN-CORE/incore-services/issues/122) [#121](https://github.com/IN-CORE/incore-services/issues/121) [#119](https://github.com/IN-CORE/incore-services/issues/119)
```

Comment:
might be cleaner if we just split them into multiple entries? Thoughts

Reply:
Yes, but I was lazy about that hahaha... I updated it now.